CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+## v1.0.0-beta.5 (2019-12-09)
+
+### Features
+
+* **config**: add a typed error on config file not found ([#264](https://github.com/scaleway/scaleway-sdk-go/pull/264))
+* **core**: get Region from Zone ([#255](https://github.com/scaleway/scaleway-sdk-go/pull/255))
```

Comment:
maybe group by subject (eg all config, then all core, etc.), to make it more readable

Reply:
done